test/unit/next-babel-loader-prod.test.ts

```diff
@@ -323,75 +323,5 @@ describe('next-babel-loader', () => {
         `var __jsx = React.createElement;import "core-js";import { bar } from "a";import baz from "b";import * as React from "react";import { yeet } from "c";import baz3, { cats } from "d";import { c, d } from "e";import { e as ee } from "f";export var __N_SSG = true;export default function () {  return __jsx("div", {    __self: this,    __source: {      fileName: _jsxFileName,      lineNumber: 1,      columnNumber: 326    }  }, cats + bar());}`
       )
     })
-
-    it('should support optional chaining for JS file', async () => {
```

Comment:
It is safe to remove those unit test cases. Next.js has an another integration test case `test/integration/optional-chaining-nullish-coalescing` to make sure a Next.js application with optional chaining and nullish operator will work.

Reply:
Let's keep these unit tests just for a sanity check